tools/preflight/preflight.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-useless-concat */
+/* eslint-disable no-alert */
+/* eslint-disable no-return-assign */
```

Comment:
A lot of these seem unnecessary. I would fix the root complains rather than opting out of all of these.

Reply:
for the alert reading the documentation seems like it is a ui issue is why it is blocked, would you prefer I create a custom alert box? if so do you have a design standard I should follow. The others I will fix.